src/js/player.js

```diff
@@ -423,6 +425,24 @@ class Player extends Component {
    * @method updateStyleEl_
    */
   updateStyleEl_() {
+    if (window.VIDEOJS_NO_DYNAMIC_STYLE === true) {
+      /* jshint ignore:start */
+      let {width_: width = this.options_.width, height_: height = this.options_.width} = this;
```

Comment:
This syntax excludes a lot of contributors from understanding what's happening here. Took me a minute. I think @mmcc is still working on it. :)